CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.5.3 (2019-05-06)
+
+- Fixed `CurrencyPipe` and `DecimalPipe` to be backwards compatible with Angular 4. [#74](https://github.com/blackbaud/skyux-core/pull/74)
```

Comment:
"Fixed `SkyNumericPipe` to include proper locale data when used with Angular version 5 and higher." (see: https://github.com/blackbaud/skyux-datetime/blob/master/CHANGELOG.md#322-2019-04-30)

Reply:
The PR that this item links to says it replaces `CurrencyPipe` and `DecimalPipe`. So saying in the changelog entry that we "Fixed" them seems a bit off to me. Did we remove them and replace them with a custom utility like the PR indicates?

Reply:
We replaced our internal-only usage of Angular's pipes, but we "fixed" the numeric pipe to work as expected on Angular versions greater than 5.

Reply:
In a way, the numeric pipe is currently broken on SKY UX 3 SPAs, which is why this is a "fix".

Reply:
@blackbaud-johnly @Blackbaud-SteveBrush words updated.